examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Inc/mems_cube_hal.h

```diff
@@ -0,0 +1,121 @@
+/**

```

Comment:
You're right. I thought I still had something necessary in that file, but I didn't. I've removed it.